mono/eglib/glib.h

```diff
@@ -1133,10 +1139,37 @@ gboolean   g_file_test (const gchar *filename, GFileTest test);
 
 gchar *g_mkdtemp (gchar *tmpl);
 
-
 /*
  * Low-level write-based printing functions
  */
+
+static inline int
+g_async_safe_fgets (char *str, int num, int handle, gboolean *newline)
+{
+	memset (str, '\0', sizeof(char) * num);
```

Comment:
`'\0'` can just be 0.
In fact the parameter is `int`, not `char`.

`sizeof (char)` == 1, no need to multiply by it.